src/Sylius/Bundle/ShopBundle/Resources/views/Email/orderConfirmation.html.twig

```diff
@@ -1,29 +1,29 @@
 {% extends '@SyliusShop/Email/layout.html.twig' %}
 
 {% block subject %}
-    Order confirmation
+    {{ 'sylius.email.order_confirmation.subject'|trans({}, null, localeCode) }}
```

Comment:
This could also be written as:
```
{% trans into localeCode %}sylius.email.order_confirmation. subject{% endtrans %}
```
What is a better option? /cc @Sylius/core-team

Reply:
I like the `{% trans into localeCode %}` version when no parameters or domain is present. It looks cleaner :D

Reply:
But it would be nice to create a PR for twig that adds a `trans_default_locale` just as they have a `trans_default_domain`. That would really help with the readability of such templates as this